tests/python/pytest_opinfos.py

```diff
@@ -1103,6 +1105,33 @@ def torch_reshape_sym_fn(input_tensor, output_shaped_tensor):
 )
 tensor_creation_ops.append(uniform_opinfo)
 
+matmul_ops = []
+
+
+def matmul_wrapper(fd, data_type, a, b):
+    t1 = fd.ops.matmul(a, b)
+    t2 = fd.ops.cast(t1, dtype=DataType.Half)
```

Comment:
`data_type` is unused. I think you mean to replace `DataType.Half` with it`.

Reply:
Thanks for catching that!